bin/createTestDB.js

```diff
@@ -237,8 +248,8 @@ async function init () {
       const rats = []
 
       logger.info('creating %d clients for %s', size.rescues, p)
-      for(let client = 0; client < size.rescues; client++) {
-        let clientName = 'client-' + p + '-' + ('' + client).padStart(3, '0')
+      for (let client = 0; client < size.rescues; client++) { // eslint-disable-line
```

Comment:
im a bit confused on how this loop works, please explain how it works and why it can't be refactored to use an alternative like a for-of while, map, etc..

Reply:
This and the other loops probably could be refactored into a for-of - I was just being lazy.  tbh there are a bunch on non-standard stuff I am doing in this script because it was just designed to seed the DB without too much fuss.  I'm planning on moving most of this code into `test/v2/support` as I review tests for each endpoint, so it might be better to just remove this file for the time being

Reply:
I'll approve the pr under the pretence that this will be improved in the future, just add some code comments I've mentioned first